src/WingetCreateTests/WingetCreateTests/UnitTests/CommonTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+
+namespace Microsoft.WingetCreateUnitTests
+{
+    using System;
+    using System.IO;
+    using Microsoft.WingetCreateCLI;
+    using NUnit.Framework;
+
+    /// <summary>
+    /// Test cases for verifying common functions for the CLI.
+    /// </summary>
+    public class CommonTests
+    {
+        /// <summary>
+        /// Tests the ability to retrieve the path for display purposes.
+        /// </summary>
+        [Test]
+        public void VerifyPathSubstitutions()
+        {
+            string path1 = Environment.GetFolderPath(Environment.SpecialFolder.UserProfile) + "\\foo\\bar\\baz";
```

Comment:
Nit: Make `"\\foo\\bar\\baz"` a local string variable inside this test method it's being for all the paths.